src/sentry/web/frontend/base.py

```diff
@@ -41,83 +34,108 @@
 
 
 class OrganizationMixin:
+    # This attribute will only be set once determine_active_organization is called.  Subclasses should likely invoke
+    # that method, passing along the organization_slug context that might exist (or might not).
+    active_organization: Optional[ApiOrganization]
```

Comment:
I decided _not_ to initialize this attribute, preferring that tests raise an `AttributeException` if they attempt to access it before calling `determine_active_organization`.  I felt that level of feedback was more useful than a NoneType (because this can legitimately be None in many cases).